application/helpers/admin/statistics_helper.php

```diff
@@ -401,7 +403,8 @@ function buildSelects($allfields, $surveyid, $language)
                 foreach ($aresult as $arow) {
                     // only add condition if answer has been chosen
                     if (in_array($arow['title'], $_POST[$pv])) {
-                        $mselects[] = Yii::app()->db->quoteColumnName(substr($pv, 1, strlen($pv)) . $arow['title']) . " = 'Y'";
+                        $fieldname = substr($pv, 1, strlen($pv)) . $arow['title'];
+                        $mselects[] = Yii::app()->db->quoteColumnName($fieldname) . " = '" . getEncryptedCondition($responseModel, $fieldname, 'Y') . "'";
```

Comment:
usage of https://www.yiiframework.com/doc/api/1.1/CDbConnection#quoteValue-detail seems better, no ?

Reply:
yeah, quoteValue. I agree.